be/src/olap/base_tablet.h

```diff
@@ -146,7 +146,8 @@ class BaseTablet {
                            RowsetSharedPtr rowset, const TupleDescriptor* desc,
                            OlapReaderStatistics& stats, std::string& values,
                            bool write_to_cache = false);
-
+    bool key_is_not_in_segment(Slice key, const KeyBoundsPB& segment_key_bounds,
```

Comment:
the method name feels a bit unnatural; it should be called `is_key_in_segment`.
And it's better as a util method, not a member method of `BaseTablet`

Reply:
Oh, I see. You can only confirm that the key is not in this segment, but you cannot confirm that it is in the segment.
You need to add comment for this method.

Reply:
Oh, I see. You can only confirm that the key is not in this segment, but you cannot confirm that it is in the segment.
You need to add comment for this method.